users.yml

```diff
@@ -1,3 +1,6 @@
 ---
 admins:
```

Comment:
List of users for test and running aplication should not be in the same file, don't merge this files.